Tests/ProjectDescriptionTests/Dependencies/Dependency+CarthageRequirementTests.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import XCTest
+
+@testable import ProjectDescription
+
+final class DependencyCarthageRequirementTests: XCTestCase {
```

Comment:
After removing the extension remember to move these tests into `DependencyTests`.